src/image.c

```diff
@@ -69,7 +69,7 @@ static int wolfBoot_verify_signature(uint8_t *hash, uint8_t *sig)
     int ret, verify_res = 0;
 #ifdef WOLFBOOT_TPM
     WOLFTPM2_KEY tpmKey;
-    #ifdef DEBUG
+    #ifdef TPM_DEBUG
```

Comment:
Suggest using the wolfTPM macro `DEBUG_TPM`. Or change to a macro specific to wolfBoot like `WOLFBOOT_DEBUG_TPM`.

Reply:
We renamed it to `WOLFBOOT_DEBUG_TPM` as suggested. We also uniformed the other define `s/X86_64_EFI_DEBUG/WOLFBOOT_DEBUG_EFI/g`